packages/ui/src/stories/Button/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import ButtonStyles from "./button.style";
+
+export interface ButtonProps {
+  type: "primary" | "secondary";
+  id: string;
+  htmlType: string;
```

Comment:
```suggestion
  htmlType: 'button' | 'submit' | 'reset'| undefined;
```

Have you tried that? It is better to be more specific here. This way in the future when a new developer tries to use this library and passes `htmlType` a string that is _not_ one of those string we define above they will see an error.

Reply:
I tried `'button' | 'submit' | 'reset'| undefined` but in our Button.stories.tsx, we are using `.add("secondary", () => <Button type="secondary">Call to Action</Button>)` so the linter throws an error.

And if I do something like 'button' | 'submit' | 'reset'| undefined | 'primary' | 'secondary' the linter throw an error because what I mentioned before:

> The expected type comes from property 'type' which is declared here on type 'DetailedHTMLProps<ButtonHTMLAttributes, HTMLButtonElement>'

Reply:
wait there are two different props, one we call it `htmlType`, and one `type`, the comments we are having is about the `htmlType`. Could that be `secondary`?

Reply:
Oooooh, I see. So in the `Interface` we could have:
`type: "primary" | "secondary";
  htmlType: "button" | "submit" | "reset" | undefined;`
  
  And then:
  
  `<button
        type={htmlType}
        {...}
        </button>`

Reply:
I really have to catch up with Typescript and Typescript in React.
Unfortunately I have German classes today, 6 hours!